test/lint/pass_lint_fix_json.sh

```diff
@@ -0,0 +1,39 @@
+#!/bin/sh
+
+set -o errexit
+set -o nounset
+
+TMP="$(mktemp -d)"
+clean() { rm -rf "$TMP"; }
+trap clean EXIT
+
+cat << 'EOF' > "$TMP/schema.json"
+{
+  "$schema": "http://json-schema.org/draft-06/schema#",
+  "type": "string",
+  "enum": [ "foo" ]
+}
+EOF
+
+"$1" lint "$TMP/schema.json" --fix --json >"$TMP/output.json" 2>&1 && CODE="$?" || CODE="$?"
+test "$CODE" = "0" || exit 1
```

Comment:
You don't need to test code exit code 0. If you don't put any of this, non-zero codes will fail out of the box. Let me give you some suggested changes

```suggestion
```